src/client/App.js

```diff
@@ -1,20 +1,30 @@
 import React from "react";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import TestComponent from "./components/TestComponent/TestComponent";
+import About from "./pages/About";
+import Home from "./pages/Home";
+import ShareMeal from "./pages/ShareMeal";
+import OurMeals from "./pages/OurMeals";
+import Navbar from "./components/Navbar";
+import Footer from "./components/Footer";
+import MealDetail from "./components/MealDetail";
 
 function App() {
   return (
+    <>
+
     <Router>
```

Comment:
Ideally, this whole "block" should be indented. I would recommend auto-formatting all your code.